chapters/de/chapter3/3.mdx

```diff
@@ -110,12 +110,12 @@ import numpy as np
 preds = np.argmax(predictions.predictions, axis=-1)
 ```
 
-Jetzt können wir diese Vorhersagen in `preds` mit den Labels vergleichen. Wir greifen auf die Metriken aus der 🤗 Bibliothek Datasets zurück, um unsere Funktion `compute_metric()` zu erstellen. Die mit dem MRPC-Datensatz verbundenen Metriken können genauso einfach geladen werden, wie wir den Datensatz geladen haben, diesmal mit der Funktion `load_metric()`. Das zurückgegebene Objekt verfügt über eine Berechnungsmethode, mit der wir die Metrik auswerten können:
+Jetzt können wir diese Vorhersagen in `preds` mit den Labels vergleichen. Wir greifen auf die Metriken aus der 🤗 Bibliothek [Evaluate](https://github.com/huggingface/evaluate/) zurück, um unsere Funktion `compute_metric()` zu erstellen. Die mit dem MRPC-Datensatz verbundenen Metriken können genauso einfach geladen werden, wie wir den Datensatz geladen haben, diesmal mit der Funktion `evaluate.load()`. Das zurückgegebene Objekt verfügt über eine Berechnungsmethode, mit der wir die Metrik auswerten können:
 
 ```py
-from datasets import load_metric
+import evaluate
 
-metric = load_metric("glue", "mrpc")
+metric = evaluate.load("glue", "mrpc")
```

Comment:
Is this the way you recommend users load metrics vs 

```
from evaluate import load

metric = load("glue", "mrpc")
```

Just wondering why we don't import the `load()` function explicitly. I'm not fussed either way, but note that whatever you choose will become the "norm" for future generations :)

Reply:
The main reason is that `load` is a pretty broad term and so in a long script it might not be very obvious where it comes from and what it does. Similar to how people usually use `json.load()` instead of `load()` (and in that case we would have a name collision).

Reply:
Thanks for clarifying! I like the analogy with `json.load()` a lot - let's keep things are you have them